src/createTwitchVideoCompiler/youtubeUploader.js

```diff
@@ -0,0 +1,192 @@
+// @flow
+
+import fs from 'fs';
+import readline from 'readline';
+import { google } from 'googleapis';
+
+export default async function youtubeUploader(
```

Comment:
"youtubeUploader" sounds like a class name more than a function name. You should rename this to "uploadToYoutube" or something.